mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(salesTax) {
+  let taxRate = 20;
+  let calculateSalesTax = salesTax * (1 + taxRate / 100);
+  return calculateSalesTax;
```

Comment:
although, it might work,
what is requiring improvement here the most is naming.
`calculateSalesTax` is the function name. you should always avoid giving the same names to variables elsewhere.
Oh, and note that variable's name should not have a verb word in it.